### DISCUSSION TITLE
Inconsistent logstash.conf file

### DISCUSSION THREAD

**Discussion by vertebarbe** [telekom-security/tpotce#1101] [Q&A] [answered]
First of all, thank you for this great project !

I recently tried to modify the logstash configuration, which worked on the previous versions.
The problem I faced is that the latest docker image of logstash (2204) does not have the version 7.2.12 of the geoip filter,
see below:
![image](https://user-images.githubusercontent.com/35434346/169526267-2fbf8784-6979-40c8-904d-fd86ea5539c6.png)

So when I base my new configuration on [this one](https://github.com/telekom-security/tpotce/tree/master/docker/elk/logstash/dist/logstash.conf), I have the following error:
![image](https://user-images.githubusercontent.com/35434346/169529960-05296282-b2a1-455b-a87f-dcc9d614a488.png)

I fixed this by replacing '12' by '11' lines 641, 646, 660 and 666 of the default logstash.conf. 
![image](https://user-images.githubusercontent.com/35434346/169532422-4f5351f9-bcc9-4e25-8568-0f0d8b17b4ce.png)
<br>
<br>
<br>

<a name="info"></a>
## ⚠️ Basic support information (commands are expected to run as `root`)

- What version of the OS are you currently using `lsb_release -a` and `uname -a`?
Debian 11, 5.10.0-14-cloud-amd64 #1 SMP Debian 5.10.113-1 (2022-04-29) x86_64 GNU/Linux
- What T-Pot version are you currently using?
Latest commit on the master branch ([f13d082](https://github.com/telekom-security/tpotce/commit/f13d08287f45219522e8f386fa6aa29eb334a8ed)).
- What edition (Standard, Nextgen, etc.) of T-Pot are you running?
Standard
- What architecture are you running on (i.e. hardware, cloud, VM, etc.)?
AWS
- Did you have any problems during the install? If yes, please attach `/install.log` `/install.err`.
No.
- How long has your installation been running?
Less than 24 hours
- Did you install upgrades, packages or use the update script?
No.
- Did you modify any scripts or configs? If yes, please attach the changes.
In **/opt/tpot/etc/tpot.yml**:
![image](https://user-images.githubusercontent.com/35434346/169531224-ed65e37e-e4c8-4670-85b4-13bd3d77ffeb.png)

**Comment by t3chn0m4g3** [comment, marked as answer]
Thanks, I converted this into discussions since this is not an issue with T-Pot itself (works as expected), since the versions from `master` have not been deployed as docker images.
